src/components/appBar/AppBar.tsx

```diff
@@ -1,34 +1,109 @@
+import { useState, useEffect, Suspense, MouseEvent } from 'react';
 import { NavLink, Outlet, useLocation } from 'react-router-dom';
-import * as React from 'react';
-import Box from '@mui/material/Box';
-import Tabs from '@mui/material/Tabs';
-import Tab from '@mui/material/Tab';
+import {
+  Container,
+  Box,
+  Tabs,
+  Tab,
+  IconButton,
+  Menu,
+  MenuItem,
+} from '@mui/material';
+import DragHandleIcon from '@mui/icons-material/DragHandle';
+
+import logo from '../../../public/logo.svg';
 
 const AppBar = () => {
   const location = useLocation();
-  const [value, setValue] = React.useState(
-    location.pathname === '/garden' ? 1 : 0
-  );
+  const [isMobile, setIsMobile] = useState(window.innerWidth <= 500);
+  const [menuAnchorEl, setMenuAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleMenuToggle = (event: MouseEvent<HTMLButtonElement>) => {
+    setMenuAnchorEl(event.currentTarget);
+  };
 
-  const handleChange = (_event: React.SyntheticEvent, newValue: number) => {
-    setValue(newValue);
+  const handleMenuClose = () => {
+    setMenuAnchorEl(null);
   };
 
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 500);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
   return (
     <>
-      <header>
-        <Box sx={{ width: '100%', bgcolor: 'background.paper' }}>
-          <Tabs value={value} onChange={handleChange} centered>
-            <Tab label="Home" component={NavLink} to="/" />
-            <Tab label="Garden" component={NavLink} to="/garden" />
-          </Tabs>
+      <Container
+        sx={{
+          maxWidth: 1320,
+          margin: '0 auto',
+          bgcolor: 'rgba(255, 255, 255, 0.4)',
+          borderRadius: 35,
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+          }}
+        >
+          <Box>
+            <img src={logo} alt="Logo" />
+          </Box>
+          {!isMobile && (
```

Comment:
1. the condition you did here is doable with one ternary, which in my humble opinion is a bit more neat
2. i would create separate components for mobile and desktop view's, just to split the code and have nice and readable little components